### ISSUE TITLE
default batch-size is too small, and the training is too slow

### ISSUE THREAD

**Issue by lipingcoding** [lightaime/deep_gcns_torch#76]
for ogbn-mol-pcba and ogbg_ppa, the default batch-size is too small, and the training is too slow, how could you finish the experiment? have you changed the batch-size, and the according other hyper parameters setting?

**Comment by lightaime** [comment]
Hi @lipingcoding. We did not change the batch size. The experiments on ogbn-mol-pcba and ogbg_ppa indeed take quite a long time to train. You can try to increase the batch size and the learning rate accordingly if you want.

**Comment by JeanKaddour** [comment]
I've noticed this issue as well, one run with the provided hyper-parameters takes almost 3 days and my GPU is heavily under-utilized. May I ask for any suggestions on how to increase the batch size and tweak other hyper-parameters accordingly (e.g. learning rate) such that I yield reasonable performance on ogbn-molpcba in a quicker time? I don't mind reproducing the exact same results from the paper or beating other methods in the leaderboard; I want to simply try out a research idea that is agnostic to the model.
